microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonDeployableContainer.java

```diff
@@ -201,9 +244,9 @@ void startServer(RunContext context, Path[] classPath)
         }
 
         context.classLoader = new HelidonContainerClassloader(parent,
-                                                              urlClassloader,
-                                                              excludedLibrariesPattern,
-                                                              containerConfig.getUserParentClassloader());
+                urlClassloader,
```

Comment:
this change should not be part of the PR

Reply:
sorry its not any more